Automattic/wp-calypso
# [Odie] Add expriment and new svg for Wapuu

## Proposed Changes

@Automattic/team-calypso-frameworks I would love your opinion about the addition of the experiment. I had to add a HoC but the other option is to add the experiment INSIDE the provider, which seems not optimal in my opinion, as the component will be loaded and not displayed, vs not loaded and displayed at all.

## Testing Instructions

1. Use this live branch
2. Assign yourself the experiment `calypso_plans_wapuu_sales_agent_v0` as treatment user
3. Navigate to upgrades -> plans
4. Assert that you can see a new chatbot in the right bottom corner that is blinking until you click on it for first time
5. Talk with the chatbot
6. Assert that the chatbox is ONLY visible in upgrades -> Plans
7. Assign yourself the experiment `calypso_plans_wapuu_sales_agent_v0` as control user
8. Navigate to upgrades -> plans
9. Assert that the chatbox is not displayed anymore

## Pre-merge Checklist
